Zadanie 17.4 Praktyczne wykorzystanie szablonów/views/after-log-page.pug

```diff
@@ -0,0 +1,9 @@
+doctype html
+html
+    head
+        title='example page after login'
+        style
+            include ../assets/style-logged.css
+    body
+        h1 Welcome to my website!
+        a(href='http://localhost:3000/') Back!
```

Comment:
zeby link dzialal na kazdym srodowisku, to nie dolaczaj adresu localhost, tylko uniwersalnie `/`